scripts/generate/mockgen.go

```diff
@@ -50,29 +50,20 @@ func main() {
 	packageName := os.Args[1]
 	interfaces := os.Args[2]
 	outputPath := os.Args[3]
-	re := regexp.MustCompile("(?m)[\r\n]+^.*" + projectVendor + ".*$")
-
+	re := regexp.MustCompile(projectVendor)
 	copyrightHeader := fmt.Sprintf(copyrightHeaderFormat, time.Now().Year())
 
 	path, _ := filepath.Split(outputPath)
 	err := os.MkdirAll(path, os.ModeDir|0755)
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
-	}
+	handleError(err)
```

Comment:
Rather than doing this with a `handleError` function, I'd instead move the bulk of the `main` function to a new function that just returns an `error`, and have all the logic you have in `handleError` just be in `main` after this new function returns.